util/src/network/session.rs

```diff
@@ -329,16 +331,19 @@ impl Session {
 			PACKET_DISCONNECT => {
 				let rlp = UntrustedRlp::new(&packet.data[1..]);
 				let reason: u8 = try!(rlp.val_at(0));
+				if self.had_hello {
```

Comment:
This is to filter out disconnects that happen before the session is established. Disconnect that happens when the session is active is much more interesting to log as it means that the peer decided to disconnect us based on our activity